lib/rojo-thinks.js

```diff
@@ -0,0 +1,73 @@
+const rojoThinks = (question) => {
+  const startWithWhen = /when.*/i;
+  const startWithWhat = /what.*/i;
+  const startWithWhere = /where.*/i;
+  const startWithWhich = /which.*/i;
+
+  const containsTime = /.+time.*/i;
+  const containsAge = /.+age.*/i;
+  const containsDay = /.+day.*/i;
+  const containsGo = /.+go.*/i;
+  const containsMy = /.+my.*/i;
+  const containsCountry = /.+country.*/i;
+  const containsCity = /.+city.*/i;
+
```

Comment:
This is begging for a refactor. Notice to pattens of words that start with `startWith` and `contains`. Make each an object and place outside the scope of `rojoThinks`.

```js
const startsWith = {
  when: /when.*/i,
  what: /what.*/i,
  ..etc
}

const contains = {
  time: /.+time.*/i,
  ...etc
}
```